inc/admin/dashboard/namespace.php

```diff
@@ -240,39 +238,33 @@ function pending_invitations_callback() {
 	if ( ! $user_has_books ) {
-		echo '<p>' . __( 'You do not have access to any books at the moment.', 'pressbooks' ) . '</p>';
+		echo '<p>' . esc_html__( 'You do not currently have access to any books on this network.', 'pressbooks' ) . '</p>';
 	}
 	$contact = \Pressbooks\Utility\main_contact_email();
 	// Values can be 'all', 'none', 'blog', or 'user', @see wp-signup.php
 	$active_signup = apply_filters( 'wpmu_active_signup', get_site_option( 'registration', 'none' ) );
 	if ( in_array( $active_signup, [ 'none', 'user' ], true ) ) {
-		echo '<p>';
-		_e( 'This network does not allow users to create new books. To create a new book, please contact your Pressbooks Network Manager', 'pressbooks' );
-		if ( ! empty( $contact ) ) {
-			echo ' ' . __( 'at', 'pressbooks' ) . " $contact";
-		} else {
-			echo '.';
+		echo '<p>' . esc_html__( 'This network does not allow users to create new books. To create a new book, please contact a network manager', 'pressbooks' );
+		if ( ! empty( $contact ) && strpos( $contact, '@pressbooks.com' ) === false ) {
+			echo ' ' . esc_html__( 'at ', 'pressbooks' ) . sprintf( '<a href="mailto:%1$s">%2$s</a>', $contact, $contact );
 		}
-		echo '</p>';
+		echo '.</p>';
 	} else {
 		$href_create = network_home_url( 'wp-signup.php' );
-		$text_create = __( 'Create a Book', 'pressbooks' );
+		$text_create = esc_html__( 'Create a book', 'pressbooks' );
 		$href_clone = admin_url( 'admin.php?page=pb_cloner' );
-		$text_clone = __( 'Clone a Book', 'pressbooks' );
-		echo "<p><a class='button button-hero button-primary' href='{$href_create}'>{$text_create}</a></p><p><a class='button button-hero button-primary' href='{$href_clone}'>{$text_clone}</a></p>";
+		$text_clone = esc_html__( 'Clone a book', 'pressbooks' );
+		echo '<p>' . sprintf( esc_html__( 'Get started on your next publishing project by creating a new book or cloning an existing book. The %1$s includes thousands of openly licensed books available for cloning.', 'pressbooks' ), sprintf( '<a href="https://pressbooks.directory" target="_blank">%s</a>', esc_html__( 'Pressbooks Directory', 'pressbooks' ) ) ) . "</p><p><a class='button button-hero button-primary create-book' href='{$href_create}'>" . $text_create . "</a></p><p><a class='button button-hero button-primary clone-book' href='{$href_clone}'>" . $text_clone . '</a></p>';
 	}
 	if ( ! $user_has_books ) {
-		echo '<p>';
-		_e( "You can also request access to an existing book by contacting the book's author or the institution's Pressbooks Network Manager", 'pressbooks' );
-		if ( ! empty( $contact ) ) {
-			echo ' ' . __( 'at', 'pressbooks' ) . " $contact";
-		} else {
-			echo '.';
+		echo '<p>' . esc_html__( 'You can also request access to an existing book by contacting your network manager', 'pressbooks' );
+		if ( ! empty( $contact ) && strpos( $contact, '@pressbooks.com' ) === false ) {
+			echo ' ' . esc_html__( 'at ', 'pressbooks' ) . sprintf( '<a href="mailto:%1$s">%2$s</a>', $contact, $contact );
```

Comment:
Same thing as above, you can use the same placeholder to avoid using the same variable twice.